.circleci/config.yml

```diff
@@ -1,5 +1,13 @@
 version: 2.0
 
+anchor_build: &anchor_build
+  name: Build
+  command: mvn -B -DtrimStackTrace=false -fae --quiet -DskipTests install
+
+anchor_unittest: &anchor_unittest
+  name: Run unit tests
+  command: mvn -B -DtrimStackTrace=false -fae --quiet -DskipITs install -P release
```

Comment:
Do we actually need anchor_build? IIUC, we always build then unittest right after, so we might as well not do anchor_build and just unittest?

Reply:
It produces a nice result on the CircleCI build page - you can see at a glance if the build passed and the unit tests failed, or if the build failed before it got to the unit tests. A failure of the build phase is a very strong signal that the pull requester messed up :-)